shell/platform/darwin/macos/framework/Source/FlutterView.h

```diff
@@ -18,7 +18,7 @@
  * backward compatibility, single-view APIs will always operate the view with
  * this ID. Also, the first view assigned to the engine will also have this ID.
  */
-constexpr uint64_t kFlutterDefaultViewId = 0;
+constexpr uint64_t kFlutterDefaultViewId = 1;
```

Comment:
Just curious, why was this changed? Does the framework have any expectations of the default view having an ID of `0`?

Reply:
Moving this discussion to https://github.com/flutter/engine/pull/38981#discussion_r1081691327 .